scheduler/src/cook/kubernetes/api.clj

```diff
@@ -421,8 +427,12 @@
             get-pod-namespaced-key
             callbacks
             (fn []
+              (prom/set prom/total-pods {:compute-cluster compute-cluster-name} (-> @all-pods-atom keys count))
               (set-metric-counter "total-pods" (-> @all-pods-atom keys count) compute-cluster-name)
-              (when max-total-pods (set-metric-counter "max-total-pods" max-total-pods compute-cluster-name)))
+              (when max-total-pods
+                (do
```

Comment:
Sorry about that, intellij was complaining about something and I thought it was because I was missing `do`, but it must have been something else. Fixed!